ui/app/adapters/auth-method.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+import { default as ApplicationAdapter, namespace } from './application';
+import { dasherize } from '@ember/string';
+import classic from 'ember-classic-decorator';
+
+@classic
+export default class AuthMethodAdapter extends ApplicationAdapter {
```

Comment:
`question`: I noticed we're not making any calls to `super` and thus we might not need to extend the `ApplicationAdapter` nor use the `classic` decorator as a result. Is it possible for us to create the adapter without it? What might be the drawbacks of my suggestion?

I believe that Adapters and Serializers may be [deprecated](https://github.com/emberjs/rfcs/pull/860) soon!

Reply:
Interesting about the deprecation! I added the `@classic` decorator in the interest of getting our tests (which swallow up adapter errors) to progress but I'll take a look at `super`ing this.

Reply:
Checked, and because we leverage `buildURL` and `ajax` methods from the application adapter, I don't think we can avoid the decorator.